dnsext-dnssec/DNS/SEC/Verify/RSA.hs

```diff
@@ -86,10 +86,10 @@ rsaVerify (alg, pkcs1) pubkey sig msg = do
      SHA-512:     (0x)30 51 30 0d 06 09 60 86 48 01 65 03 04 02 03 05 00 04 40 || H.
  -}
 sha1   :: (SHA1,   ByteString)
-sha1    = (SHA1,   BS.pack [0x30, 0x21, 0x30, 0x09, 0x06, 0x05, 0x2b, 0x0e, 0x03, 0x02, 0x1a, 0x05, 0x00, 0x04, 0x14])
+sha1    = (SHA1,   B8.pack "\x30\x21\x30\x09\x06\x05\x2b\x0e\x03\x02\x1a\x05\x00\x04\x14")
```

Comment:
`B8.pack` is not necessary!

Reply:
https://github.com/kazu-yamamoto/dnsext/pull/21/commits/05b9839a16f2ca714419533fe81aca2d06903427